datasets/aime-2024/prepare.py

```diff
@@ -0,0 +1,147 @@
+import json
```

Comment:
License header is missing. Plus if you used some other codebases to get part of this logic, we should reference them (if you wrote it all fully yourself, then just the license)

Reply:
ok, i will add